src/tasks/MonoTargetsTasks/NetTraceToMibcConverterTask/NetTraceToMibcConverter.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Concurrent;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.Text.Json.Serialization;
+
+
+public class NetTraceToMibcConverter : ToolTask
+{
+    /// <summary>
+    /// Entries for assemblies referenced in a .nettrace file. Important when you run traces against an executable on a different machine / device
+    /// </summary>
+    [Required]
+    public ITaskItem[] Assemblies { get; set; } = Array.Empty<ITaskItem>();
+
+    /// <summary>
+    /// NetTrace file to use when invoking dotnet-pgo for
+    /// </summary>
+    [Required]
+    public string NetTracePath { get; set; } = "";
+
+    /// <summary>
+    /// Directory where the mibc file will be placed
+    /// </summary>
+    [NotNull]
+    [Required]
+    public string? OutputDir { get; set; }
+
+    /// <summary>
+    /// The path to the mibc file generated from the converter.
+    /// </summary>
+    [Output]
+    public string? MibcProfilePath { get; set; }
```

Comment:
```suggestion
    public string? MibcFilePath { get; set; }
```

Reply:
This output should conform to the associated input of `MonoAOTCompiler` in name, and perhaps type?